### ISSUE TITLE
Fails to parse feed which has a whitespace in rss-version attribute (example provided)

### ISSUE THREAD

**Issue by p-meier** [rometools/rome#185]
### Problem

I got this feed which is not parseable (notice the whitespace in the rss-version attribute!): http://www.chemicalonline.com/rssfeeds/RssNewsletter

```xml
<?xml version="1.0" encoding="utf-8" ?>
<rss version=" 2.0" xmlns:media="http://search.yahoo.com/mrss/">
    <channel>
        <title>Chemical Online Newsletters</title>
...
```

Due to this there is no parser selected (is null) in ```WireFeedInput``` which results then in the IllegalArgumentException:

```java
...
 public WireFeed build(Document document) throws IllegalArgumentException,FeedException {
        WireFeedParser parser = getFeedParsers().getParserFor(document);
        if (parser==null) {
            throw new IllegalArgumentException("Invalid document");
        }
        return parser.parse(document, _validate);
    }
...
```

### Possible Solution

In ```RSS20Parser``` where version is checked with ```startsWith()```the version could be trimmed before the check. By doing this it would work as expected. Maybe this should be done in all parsers to resolve this issue not only for rss.

Here the is the code I talked about above: 
```java
...
@Override
    public boolean isMyType(final Document document) {
        final Element rssRoot = document.getRootElement();
        final Attribute version = rssRoot.getAttribute("version");
        // as far ROME is concerned RSS 2.0, 2.00 and 2.0.X are all the same, so let's use
        // startsWith for leniency.
        return rssRoot.getName().equals("rss") && version != null && version.getValue().startsWith(getRSSVersion());
    }
...
```

**Comment by mishako** [comment]
I agree that Rome should be lenient in this case.

Fixed in #213 @snoopdave @PatrickGotthard 
Haven't touched other parsers because I think we should deprecate all other versions (RSS 2.0 has been released 13 years ago).

**Comment by mishako** [comment]
@p-meier Rome 1.6.0 has been released with this issue fixed.
